manifests/0000_12_etcd-operator_08_etcdquorumguard_deployment.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: openshift-etcd
   annotations:
     include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
     exclude.release.openshift.io/internal-openshift-hosted: "true"
 spec:
   replicas: 3
```

Comment:
I would have figured that you would not want quorum-guard for single node?

Reply:
I see that you will followup.

Reply:
yes - that one we expect to disable